src/metrics/index.js

```diff
@@ -19,33 +19,56 @@ limitations under the License.
 'use strict';
 
 const log = require('../utils/logger')('splitio-metrics');
-const tracker = require('../utils/logger/timeTracker');
+const tracker = require('../utils/timeTracker');
 
 const repeat = require('../utils/fn/repeat');
 
 const metricsService = require('../services/metrics');
-const metricsServiceRequest = require('../services/metrics/post');
+const metricsTimesServiceRequest = require('../services/metrics/times');
+const metricsCountersServiceRequest = require('../services/metrics/counters');
 const metricsDTO = require('../services/metrics/dto');
 
 const impressionsService = require('../services/impressions');
 const impressionsBulkRequest = require('../services/impressions/bulk');
 const impressionsDTO = require('../services/impressions/dto');
 
+const {
+  SegmentChangesMetrics,
+  SplitChangesMetrics,
+  MySegmentsMetrics,
+  SDKMetrics
+} = require('./Logs');
+
 const MetricsFactory = (settings: Object, storage: SplitStorage): Startable => {
   const pushMetrics = (): Promise<void> => {
     if (storage.metrics.isEmpty()) return Promise.resolve();
 
     log.info('Pushing metrics');
     tracker.start(tracker.C.METRICS_PUSH);
 
-    return metricsService(metricsServiceRequest(settings, {
-      body: JSON.stringify(metricsDTO.fromGetTreatmentCollector(storage.metrics))
-    }))
-    .then(() => {
-      tracker.stop(tracker.C.METRICS_PUSH);
-      return storage.metrics.clear();
-    })
+    // POST latencies
+    const latenciesPromise = metricsService(
+      metricsTimesServiceRequest(settings, {
+        body: JSON.stringify(metricsDTO.fromLatenciesCollector(storage.metrics))
+      }
+    ))
+    .then(() => storage.metrics.clear())
     .catch(() => storage.metrics.clear());
+
+    // POST counters
+    const countersPromise = metricsService(
+      metricsCountersServiceRequest(settings, {
+        body: JSON.stringify(metricsDTO.fromCountersCollector(storage.count))
+      }
+    ))
+    .then(() => storage.count.clear())
+    .catch(() => storage.count.clear());
+
+    return Promise.all([latenciesPromise, countersPromise]).then(resp => {
+      // After both finishes, track the end and return the results
+      tracker.stop(tracker.C.METRICS_PUSH);
```

Comment:
We'll need to follow up on this because I'm not following. This is one of the times we don't track as metric, but we output to the console. 

<img width="437" alt="captura de pantalla 2017-08-02 a la s 17 27 01" src="https://user-images.githubusercontent.com/5522668/28894292-c1bd4cc4-77aa-11e7-89f7-2102cb6be0bb.png">